src/Sql/dbo/Stored Procedures/Organization_Update.sql

```diff
@@ -9,23 +9,23 @@
     @BusinessCountry VARCHAR(2),
     @BusinessTaxNumber NVARCHAR(30),
     @BillingEmail NVARCHAR(256),
-    @Plan NVARCHAR(50),
-    @PlanType TINYINT,
-    @Seats INT,
+    @Plan NVARCHAR(50) = NULL,
+    @PlanType TINYINT = NULL,
+    @Seats INT = 0,
```

Comment:
Use `NULL` here, and `COALESCE()` in the table. This will prevent the new code from setting `0` when it doesn't have a value, and keep the data.

Reply:
Done 👍